webpack/assets/javascripts/react_app/components/AuditsList/index.js

```diff
@@ -52,7 +52,18 @@ const renderResourceLink = (auditTitle, auditTitleUrl, id) => {
   return auditTitle;
 };
 
-const AuditsList = ({ data: { audits } }) => (
+const renderRequestUuidLink = (url, title, id) => (
+  <SearchLink
+    url={url}
+    textValue={title}
+    title={__(
+      'HTTP request UUID, clicking will filter audits for this request. It can also be used for searching in application logs.'
+    )}
+    id={id}
+  />
+);
+
+const AuditsList = ({ data: { audits, isOrgEnabled, isLocEnabled } }) => (
```

Comment:
Looks like bad rebase, the `isOrgEnabled` and `isLocEnabled` parameters have been removed in c949b20a5d64a906eb6d9289755de33463c4e199